tools/accuracy_checker/accuracy_checker/annotation_converters/icdar.py

```diff
@@ -119,6 +119,8 @@ def convert(self, check_content=False, progress_callback=None, progress_interval
                 text_annotation = text_area.split(',')
```

Comment:
Please update above line as 
for text_area in read_txt(gt_file, encoding='utf-8'):

Reply:
done